test/SlackMessage.spec.js

```diff
@@ -134,42 +134,33 @@ describe("SlackMessage", () => {
   });
 
   it("Sends a message to slack channel when bot does not have access to channel history", async () => {
-    const message = new SlackMessage("TOKEN", {
+    const gateway = new SlackGateway();
+    const unableToAccessError = new Error(`Unable to fetch history in channel`);
+    unableToAccessError.type = "NO_HISTORY_ACCESS";
+    unableToAccessError.channel = "1234.1234";
+
+    gateway.getStub("fetchMessage").rejects(unableToAccessError);
```

Comment:
In order to set this up I had to copy this error initialization out of the SlackGateway class which is a good hint that having it buried in there like that is a Poor Design and I will make a note to come back to this in the future to deal with it